README.md

```diff
@@ -307,6 +307,18 @@ ckan
 
 ```
 
+## Mock harvest source
+
+The docker compose configurations start an instance of the static [mock harvest source](https://github.com/alphagov/ckan-mock-harvest-sources)
+which should make it easy to populate the database with some basic standard content. To use this, add a harvest
+source with the address `http://static-mock-harvest-source:11088` and of type "CKAN". The content served by this
+harvest source is live modifiable from the `src/<version>/ckan-mock-harvest-sources/static/responses/` directory.
+
+For this harvest source to appear to work _completely_ right, you may need to add the hostname
+`static-mock-harvest-source` to your host machine's `/etc/hosts` file as an alias to `127.0.0.1`. This is because
```

Comment:
I can't connect to port 11088 on 127.0.0.1, is there something else missing?

Reply:
Hmmmmm... that's a bit troubling...

Reply:
Double checked, definitely working for me. Getting any messages from the app in your docker-compose log?

Reply:
Should this url be working - http://127.0.0.1:11088/mock-third-party/example-dataset-1/about? as its 404ing

Reply:
Uhm no, it's not something I mocked - where is that linked to from?

Reply:
It's when you view the dataset, we can just leave it for now as I think the basics work, not all the endpoints need to be available as the point of the mock harvest source was to provide an initial data source to harvest from and then build on that.

Reply:
That's interesting because I thought the actual source files were expected to be just hosted on a bog standard web server - I certainly saw a bunch of examples where the data files were just in some `wp-content/` directory somewhere...